<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>263</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>532</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Esp32 Download Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>253</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pieFlashBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>Test Flash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custFlashBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>Customer Flash</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="flashProcessBar">
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>251</width>
         <height>25</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Result</string>
       </property>
       <property name="buddy">
        <cstring>resultTextBrowser</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="resultTextBrowser">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>251</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Test Firmware</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pieFwEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="pieFwOptBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Customer Firmware</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="custFwEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="custFwOptBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>213</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;PORT</string>
         </property>
         <property name="buddy">
          <cstring>portComBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portComBox">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="searchPortBtn">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>setting</string>
    </property>
    <addaction name="actOriFw"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="setPieFwDir">
   <property name="text">
    <string>Pie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+P</string>
   </property>
  </action>
  <action name="setCoutFwDir">
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actOriFw">
   <property name="text">
    <string>OriginFirmware</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>resultTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
